frontend/src/components/ParticipantInfo.js

```diff
@@ -0,0 +1,544 @@
+  const [lastName, setLastName] = React.useState("")
+  const [dateOfBirth, setDateOfBirth] = React.useState("")
+  const [ppId, setPPId] = React.useState("")
+  const [race, setRace] = React.useState("")
+  const [gender, setGender] = React.useState("")
+  const [hasInsurance, setHasInsurance] = React.useState(false)
+  const [insuranceType, setInsuranceType] = React.useState("")
+  const [program, setProgram] = React.useState("")
+  const [service, setService] = React.useState("")
+  const [priority, setPriority] = React.useState("")
+  const [note, setNote] = React.useState("")
+  const history = useHistory()
+
+  let participant = participantStore.getParticipant()
+  let data = participantStore.getParticipantsList()
+  let participantIndex = data.findIndex(val => val.pp_id === participant.pp_id)
+  // useEffect is a hook that gets called after every render/re-render.  Empty array second argument prevents it from running again.
+  useEffect(() => {
+    if (participantIndex > -1) {
+      // eslint-disable-next-line no-console
+      // console.log(this.props.participantData)
+      // assign incoming participant data if available
+      setId(data[participantIndex].id)
+      setSSN(data[participantIndex].last_four_ssn)
+      setStartDate(data[participantIndex].start_date)
+      setInsurer(data[participantIndex].insurer)
+      setFirstName(data[participantIndex].first_name)
+      setLastName(data[participantIndex].last_name)
+      setDateOfBirth(data[participantIndex].date_of_birth)
+      setPPId(data[participantIndex].pp_id)
+      setRace(data[participantIndex].race)
+      setGender(data[participantIndex].gender)
+      setHasInsurance(data[participantIndex].is_insured)
+      // setInsuranceType(data.insuranceType)
+      // setProgram(data.program)
+      // setService(data.service)
+      // setPriority(data.priority)
+      // setNote(data.note)
+    }
+  }, [])
+
+  const handleFNameChange = e => setFirstName(e.target.value)
+  const handleLNameChange = e => setLastName(e.target.value)
+  const handleDOBChange = e => setDateOfBirth(e.target.value)
+  const handleUUIDChange = e => {
+    setPPId(e.target.value)
+    setSSN(e.target.value.slice(-4))
+  }
+  const handleRaceChange = e => setRace(e.target.value)
+  const handleGenderChange = e => setGender(e.target.value)
+  const handleHasInsuranceChange = e => setHasInsurance(e.target.value)
+  const handleInsuranceTypeChange = e => setInsuranceType(e.target.value)
+  const handleProgramChange = e => setProgram(e.target.value)
+  const handleServiceChange = e => setService(e.target.value)
+  const handlePriorityLevelChange = e => setPriority(e.target.value)
+  const handleNoteChange = e => setNote(e.target.value)
+
+  const createStartDate = () => {
+    var now = new Date()
+    var y = now.getFullYear()
+    var m = now.getMonth() + 1
+    var d = now.getDate()
+    return (
+      "" + y + "-" + (m < 10 ? "0" : "") + m + "-" + (d < 10 ? "0" : "") + d
+    )
```

Comment:
Could we use a third-party library like [`date-fns`](https://date-fns.org/v2.8.1/docs/format) for this? With date-fns this could be simplified to:

```javascript
import { format } from "date-fns"

...

const createStartDate = () => {
  return format(new Date(), 'yyyy-MM-dd')
}

```

I think it'll be helpful to have something like this to handle dates in the future as well

Reply:
Heck yea, I'll had this and re-push

Reply:
Nice